.eslintignore

```diff
@@ -1,4 +1,5 @@
 /webapp/public/**
 /webapp/javascript/util/**
 /webapp/__tests__/**
+/scripts/**
```

Comment:
had to add this because otherwise I think linter uses webpack config which has fallbacks for `fs` and that breaks everything. at least that's how I understand the issue